Flaky test AsyncUtilsTest.withTimeout_applicableTimeout

```
Error:  Tests run: 8, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.534 s <<< FAILURE! -- in io.fabric8.kubernetes.client.utils.AsyncUtilsTest
Error:  io.fabric8.kubernetes.client.utils.AsyncUtilsTest.withTimeout_applicableTimeout -- Time elapsed: 0.427 s <<< ERROR!
org.awaitility.core.ConditionTimeoutException: Condition with Lambda expression in io.fabric8.kubernetes.client.utils.AsyncUtilsTest was not fulfilled within 101 milliseconds.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.CallableCondition.await(CallableCondition.java:78)
	at org.awaitility.core.CallableCondition.await(CallableCondition.java:26)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1160)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1129)
	at io.fabric8.kubernetes.client.utils.AsyncUtilsTest.withTimeout_applicableTimeout(AsyncUtilsTest.java:69)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
	... 8 more
```

--- a/kubernetes-client-api/src/test/java/io/fabric8/kubernetes/client/utils/AsyncUtilsTest.java
+++ b/kubernetes-client-api/src/test/java/io/fabric8/kubernetes/client/utils/AsyncUtilsTest.java
@@ -66,7 +66,10 @@ class AsyncUtilsTest {
   void withTimeout_applicableTimeout() {
     final CompletableFuture<Void> future = new CompletableFuture<>();
     withTimeout(future, Duration.ofMillis(1));
-    Awaitility.await().atMost(Duration.ofMillis(101)).until(future::isDone);
+    Awaitility.await()
+        .pollInterval(Duration.ofMillis(10))
+        .atMost(Duration.ofSeconds(5))
+        .until(future::isDone);
     assertThatThrownBy(() -> future.getNow(null))
         .isInstanceOf(CompletionException.class)
         .hasCauseInstanceOf(TimeoutException.class);

